src/linear_algebra/matrix.lean

```diff
@@ -294,8 +313,49 @@ end
 
 end comp
 
-section det
+end matrix
+
+section is_basis_to_matrix
+
+variables {ι ι' R M : Type*} [fintype ι] [decidable_eq ι]
+          [comm_ring R] [add_comm_group M] [module R M]
+
+open function matrix
+
+/-- From a basis `e : ι → M` and a family of vectors `v : ι → M`, make the matrix whose columns
+are the vectors `v i` written in the basis `e`. -/
+def is_basis.to_matrix {e : ι → M} (he : is_basis R e) (v : ι → M) : matrix ι ι R :=
```

Comment:
Would it be hard to make this a linear map (or even a linear equivalence)?

Reply:
I did it. This required a bit of refactoring and some API holes filling.